Dockerfile.mysql

```diff
@@ -2,13 +2,7 @@
 FROM mysql:latest
 
 # Environment variables for MySQL configuration
-ENV MYSQL_ROOT_PASSWORD=1234
-ENV MYSQL_DATABASE=app_db
-ENV MYSQL_USER=app_user
-ENV MYSQL_PASSWORD=1234
-
-# Expose port 3306 for MySQL connections
-EXPOSE 3306
-
-# Set the data directory as a volume
-VOLUME /var/lib/mysql
+ENV MYSQL_ROOT_PASSWORD="default"
+ENV MYSQL_DATABASE="default"
```

Comment:
The `MYSQL_DATABASE` variable is set to 'default'. Ensure this value is appropriate for your application or provide instructions on how it should be overridden during deployment.